src/helpers/gutter/gutter.js

```diff
@@ -9,25 +9,17 @@ import { themeProvider } from '../../theme';
 const { theme } = themeProvider;
 
 export const gutterWidth = (props, breakpoint) => {
-  // Does the gutter prop exists?
   if (has(props, 'gutter')) {
-    // is the gutter a responsive object?
-    if (isObject(props.gutter)) {
-      // does the object actually include the current mapped breakpoint?
-      if (has(props, `gutter.${breakpoint}`)) {
-        // return the value within the gutter object
-        return props.gutter[breakpoint];
-      }
-      // if prop does not exist in gutter object use theme default
-      // TODO make function for getting theme default
-      return isObject(theme(props).gutter) ? theme(props).gutter[breakpoint] : theme(props).gutter;
+    if (isObject(props.gutter) && has(props, `gutter.${breakpoint}`)) {
```

Comment:
This does not quite cover the scenario if the gutter is an object but the current breakpoint does not exist within in it (for example when you just use the `lg` key and none of the others) this will need to then grab it from the theme as on line 20